framework/db/resource_manager.py

```diff
@@ -1,12 +1,14 @@
+import os
+import logging
+
 from framework.db import models
 from framework.config import config
 from framework.dependency_management.dependency_resolver import BaseComponent
 from framework.dependency_management.interfaces import ResourceInterface
 from framework.lib.general import cprint
-import os
-import logging
 from framework.utils import FileOperations
 
+
 class ResourceDB(BaseComponent, ResourceInterface):
```

Comment:
Indent this line? - https://github.com/hTrap/owtf/blob/pep8-fixes/framework/db/plugin_manager.py#L239